cmd/containerd/main.go

```diff
@@ -342,12 +339,13 @@ func loadSnapshotter(store content.Store) (snapshot.Snapshotter, error) {
 		}
 		sn, err := sr.Init(ic)
 		if err != nil {
-			return nil, err
+			log.G(global).WithError(err).Warnf("could not load snapshot plugin %q", name)
```

Comment:
We should add a more specific error here for snapshotter not supported. We can still output the not supported as an info message. The failure on initialize could be more serious and on a snapshotter which was expected to be used, we will want to be able to fail in that case.